dune-configurator.opam

```diff
@@ -18,7 +18,7 @@ homepage: "https://github.com/ocaml/dune"
 doc: "https://dune.readthedocs.io/"
 bug-reports: "https://github.com/ocaml/dune/issues"
 depends: [
-  "dune" {>= "2.6.0"}
+  "dune" {>= "2.7" & >= "2.6.0"}
```

Comment:
Seems like there's something wonky in how we generate dune constraints.

Reply:
Indeed